specification/alertsmanagement/resource-manager/readme.md

```diff
@@ -83,6 +94,7 @@ input-file:
   - Microsoft.AlertsManagement/preview/2019-05-05-preview/SmartGroups.json
   - Microsoft.AlertsManagement/preview/2023-01-01-preview/AlertRuleRecommendations.json
   - Microsoft.AlertsManagement/preview/2023-05-01-preview/AlertProcessingRules.json
+  - Microsoft.AlertsManagement/preview/2023-07-12-preview/AlertsManagement.json
```

Comment:
I'm not really an expert but this looks a little confusing to me. You're referring to the new preview api version from your old stable version tag, retroactively? I don't really understand why its being done this way.

Would this not cause new generation of the 'stable' package-2023-03 to include preview apis?

Reply:
(In general I would have thought that including preview api jsons in a 'stable' tag is an anti-pattern)

Reply:
This is how we've done this so far.. Do you have any other instructions of what to do?